src/components/NavBar/NavBar.style.ts

```diff
@@ -111,12 +111,15 @@ export const MenuLabel = styled.span`
 // Location page:
 export const GridContainer = styled.div`
   display: grid;
+  // On mobile, we have 4 columns: back, logo, search, menu
+  // But note that after you scroll, the search bar expands to use the logo column as well.
   grid-auto-columns: min-content auto auto min-content;
   align-items: center;
   width: 100%;
 
   @media (min-width: ${mobileBreakpoint}) {
-    grid-auto-columns: min-content min-content 1fr min-content min-content;
+    // On desktop, we have 5 columns: back, logo, search, donate, menu
+    grid-auto-columns: min-content min-content 1fr auto min-content;
```

Comment:
I noticed the donate button div was getting a bit truncated (in particular the hover background effect was smaller than the button, see below). Not sure if this is new or not, but I seemingly fixed it by changing the column width from `min-content` to `auto`. Not 100% sure if this is the best fix, but I think it works.

![image](https://user-images.githubusercontent.com/206364/132424300-01e5b85c-9732-46a2-8515-1c2625fe4a99.png)